CHANGELOG.md

```diff
@@ -1,7 +1,31 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-## 2.11.0 (TBD)
+## 2.11.1 (TBD)
+### AWSXRayRecorder.Core (2.11.1)
+- Bumped version to address package change
```

Comment:
Are you bumping the version because they have to be released together? Maybe we can say this instead?

```suggestion
- No changes. Bumping to release with lateral related packages.
```

or simply


```suggestion
- No changes.
```

Reply:
Yeah. Our packages need to be released together for the versioning to be consistent.